### DISCUSSION TITLE
jq selector string converter

### DISCUSSION THREAD

**Discussion by God-damnit-all** [TomWright/dasel#323] [General]
dasel looks like it's a lot easier to operate than jq, but the main problem I'm having is that, well, I'm already used to jq and I have a lot of jq selector strings in my notes already.

How possible would it be to convert jq selector strings? I understand that some commands wouldn't be convertible, but informing the user that one of the selectors does not yet have a dasel equivalent would be a convenience in and of itself.

**Comment by TomWright** [comment]
Thanks for the feedback. That was a key point of dasel, I wanted to selectors to be more easily understood at a glance without necessarily having to learn a bunch of shorthand.

I like the idea of this. It feels like some JQ comparison docs are in order. I'm happy to write them up but before doing so would like to collect some JQ commands here so I to show the dasel version.
